storage/sql_migrations/007_wallet.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+create table wallet_credential
+(
+    holder_did    varchar(500) not null,
+    credential_id varchar(500) not null,
+    primary key (holder_did, credential_id),
+    constraint fk_wallet_credential foreign key (credential_id) references credential (id)
```

Comment:
holder_did is a did that should be managed by this node. Should there be a foreign key on this too? Or why not?

Reply:
Probably, yes. Unless we start issuing from did:jwk's in the future, but then we'll change it again.